freqtrade/leverage/liquidation_price.py

```diff
@@ -11,7 +11,13 @@ def liquidation_price(
     leverage: float,
     trading_mode: TradingMode,
     collateral: Optional[Collateral],
-    margin_mode: Optional[MarginMode]
+    wallet_balance: Optional[float],
+    maintenance_margin_ex_1: Optional[float],
+    unrealized_pnl_ex_1: Optional[float],
+    maintenance_amount_both: Optional[float],
+    position_1_both: Optional[float],
+    entry_price_1_both: Optional[float],
+    maintenance_margin_rate_both: Optional[float]
```

Comment:
lets take off the both

Reply:
ok but needs to be added back when hedge mode is there.

Reply:
Removed here https://github.com/freqtrade/freqtrade/pull/5576/commits/e5a9e118990609b925524d640c2a288921c8a17f